src/lib/fractal.ts

```diff
@@ -112,6 +112,7 @@ export async function refetchKycStatus(
     return kycStatus;
   } catch (error) {
     // TODO: sentrify
+
```

Comment:
noticed the TODO comment above this line, did you mean to actually sentrify this part or just a formatting empty line?

Reply:
@chapati23 So this is where @denviljclarke will be continuing the intergrations